specifications/cdsc-wg1-02.md

```diff
@@ -1 +1,799 @@
+
+Directory Entry objects are formatted as JSON objects and contain the following named values:
+
+* `client_id` - From the [Client object](#client-format)
+* `client_name` - From the [Client object](#client-format)
+* `client_uri` - From the [Client object](#client-format)
+* `tos_uri` - From the [Client object](#client-format)
+* `policy_uri` - From the [Client object](#client-format)
+* `logo_uri` - From the [Client object](#client-format)
+* `scope` - From the [Client object](#client-format), containing the list of scopes approved for production use, and NOT containing the `client_admin` scope.
+* `profile_visibility` - From the [Client Settings object](#client-settings-format)
+* `profile_uri` - From the [Client Settings object](#client-settings-format)
+* `profile_description` - From the [Client Settings object](#client-settings-format)
+* `profile_button_uri` - From the [Client Settings object](#client-settings-format)
+* `profile_button_style` - From the [Client Settings object](#client-settings-format)
+* `profile_contact_name` - From the [Client Settings object](#client-settings-format)
+* `profile_contact_email` - From the [Client Settings object](#client-settings-format)
+* `profile_contact_phone` - From the [Client Settings object](#client-settings-format)
+* `profile_contact_website` - From the [Client Settings object](#client-settings-format)
+
+### 10.2. Listing Directory Entries <a id="directory-list" href="#directory-list" class="permalink">🔗</a>
+
+Clients may request to list Directory Entry objects that they have access to by making an HTTPS `GET` request, authenticated with a valid Bearer `access_token` scoped to the `client_admin` scope, to the `cds_directory_api` URL included in the [Client Registration Response](#registration-response) or [Client object](#client-format). The Client listing request responses are formatted as JSON objects and contain the following named values.
+
+* `entries` - _Array[[DirectoryEntry](#directory-entry-format)]_ - (REQUIRED) A list of Directory Entry objects for Clients who have set their `profile_visiblity` to `listed`. If no Clients are listed, this value is an empty list (`[]`). If more than 100 Directory Entry objects are available to be listed, Servers MAY truncate the list and use the `next` value to link to the next segment of the list of Directory Entry objects.
+* `next` - _URL_ or `null` - Where to request the next segment of the list of Directory Entry objects. If no next segment exists (i.e. the requester is at the end of the list), this value is `null`.
+* `previous` - _URL_ or `null` - Where to request the previous segment of the list of Directory Entry objects. If no previous segment exists (i.e. the requester is at the front of the list), this value is `null`.
+
+### 10.3. Publicly Accessible Web Directory <a id="public-directory" href="#public-directory" class="permalink">🔗</a>
```

Comment:
Is there any reason why the publicly accessible directory be in web based? Why can't it also be an API and allow the utility to determine what medium is best for their customers?